attendance.md

```diff
@@ -4,13 +4,13 @@
 
 You will take your own attendance through Garnet while on the `GA-Guest` wifi network. Please check [the graduation requirements for arriving on-time](graduation-requirements.md#attendance)!
 
-If today begins at 9:00AM, then any attendance taken at 9:01AM will be considered "tardy". Attendance cannot be taken three hours after the start time. That is: if you try to take attendance at 12:01PM, you will be considered "absent".
+If today begins at 9:00 am, then any attendance taken at 9:01 am will be considered "tardy". Attendance cannot be taken three hours after the start time. That is, if you try to take attendance at 12:01 pm, you will be considered "absent".
 
 **There are no exceptions, even if you forgot and the instructors saw you were here.**
 
 #### Checkpoints
 
-So as not to disrupt their peers, tardy students should wait until the 10 am morning lesson to enter the classrooms. The ONLY exception to this policy is when a checkpoint is being administered. If a student is up to 10 minutes late, they're asked to take the quiz in their assigned classroom. Otherwise, the checkpoint should be completed during lunch and the PR should be submitted no later than 2:30 pm. Tardies will still apply, but credit will be received for taking the checkpoint.
+As not to disrupt their peers, tardy students should wait until the 10 am morning lesson to enter the classrooms. The ONLY exception to this policy is when a checkpoint is being administered. If a student is up to 10 minutes late, they're asked to take the quiz in their assigned classroom. Otherwise, the checkpoint should be completed during lunch and the PR should be submitted no later than 2:30 pm. Tardies will still apply, but credit will be received for taking the checkpoint.
```

Comment:
"they're asked" --> "they will be asked"

Not sure, but sounds kinda weird as is, right? Could be technically correct.